doc/ref_model/chapters/chapter07.md

```diff
@@ -507,3 +519,191 @@ Table 7-2 shows security capabilities
     tools
 -   Resiliency tests run (such as hardware failures or power failure
     tests)
+
+<a name="7.11"></a>
+## 7.11 Consolidated Security Requirements
+
+Please note that **Virtual Resource Manager (VRM)** is used to collectively represent VIM, K8s, etc.
+
+<a name="7.11.1"></a>
+### 7.11.1. System Hardening
+
+|  Ref | Requirement  |  Unit  | Definition  |
+|-------|------|------|-------|
+| sec.gen.001 | The Platform **must** maintain the state to what it is specified to be and does not change unless through change management process. |   |  |
+| sec.gen.002 | All systems part of NFVI **must** support password hardening (strength and rules for updates (process), storage and transmission, etc.) |  | Hardening: NIST SP 800-63B |
+| sec.gen.003 | All servers part of NFVI **must** support a root of trust and secure boot |  |  |
+| sec.gen.004 | The Operating Systems of all the servers part of NFVI **must** be hardened |  | NIST SP 800-123 |
+| sec.gen.005 | The Platform **must** support Operating System level access control |  | Details on OS |
+| sec.gen.006 | The Platform **must** support Secure logging |  | Details |
+| sec.gen.007 | All servers part of NFVI **must** be Time synchronized with authenticated Time service |  | |
+| sec.gen.008 | All servers part of NFVI **must** be regularly updated to address security vulnerabilities |  | |
+| sec.gen.009 | The Platform **must** support Software integrity protection and verification |  | |
+| sec.gen.010 | The NFVI **must** support Secure storage (all types) |  | Expand/Delete based on other reqts |
+| sec.gen.011 | The NFVI **should** support Read and Write only storage partitions (write only permission to one or more authorized actors) |  | NIST reference? |
+| sec.gen.012 | The Operator **must** ensure that only authorized actors have physical access to the underlying infrastructure. |  | |
```

Comment:
sec.gen.012 is a physical aspect related to access. Does it needs to be a part of RM ?

Reply:
@msidana In terms of security it is critically important and is equally applicable to all implementations. Hence, the decision to include after some discussion.